Extractor/Nodes/BaseUANode.cs

```diff
@@ -562,6 +562,36 @@ protected Dictionary<string, string> BuildMetadataBase(Dictionary<string, string
             return result;
         }
 
+        protected Dictionary<string, object> BuildMetadataJsonBase(Dictionary<string, object>? extras, IUAClientAccess client, bool reversibleJson = true)
+        {
+            var result = extras ?? new Dictionary<string, object>();
+
+            if (Properties == null) return result;
+
+            foreach (var prop in Properties)
+            {
+                if (prop != null && !string.IsNullOrEmpty(prop.Name))
+                {
+                    if (prop is UAVariable variable)
+                    {
+                        result[prop.Name] = client.TypeConverter.ConvertToJsonObject(
```

Comment:
You will want to use `ConvertToJson` here, not `JsonObject`, I think. In this case we do not need to wrap every value in an object.